yaml/xpu_functions.yaml

```diff
@@ -45,6 +45,13 @@ supported:
   - remainder_.Tensor
   - remainder.Tensor_out
   - remainder.Scalar_Tensor
+  - xlogy.OutScalar_Self
+  - xlogy.OutScalar_Other
+  - xlogy.OutTensor
+  - xlogy.Scalar_Other
+  - xlogy.Scalar_Self
+  - xlogy.Tensor
+  - xlogy_.Scalar_Other
```

Comment:
ditto
please double check the auto gen file build/aten/src/ATen/RegisterCUDA.cpp for which operator should be registered
```
m.impl("xlogy.Tensor", TORCH_FN(wrapper_CUDA_xlogy_Tensor));
m.impl("xlogy.OutTensor", TORCH_FN(wrapper_CUDA_xlogy_out_OutTensor));
m.impl("xlogy_.Tensor", TORCH_FN(wrapper_CUDA_xlogy__Tensor));
```